Scotty.sh

```
@@ -0,0 +1,90 @@
+    ln -nfs $PERSISTENT_DIR/storage storage
+}
+
+# @task on:remote
+migrateDatabase() {
+    cd $NEW_RELEASE_DIR
+    php artisan migrate --force
+}
+
+# @task on:remote
+blessNewRelease() {
+    ln -nfs $NEW_RELEASE_DIR $CURRENT_DIR
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)